src/omnipool/tests.rs

```diff
@@ -450,6 +451,7 @@ fn calculate_remove_liquidity_should_work_when_correct_input_provided() {
         &position,
         imbalance,
         total_hub_reserve,
+        FixedU128::zero(),
```

Comment:
as i understand the fee wont be ever 0 again, we should test with realistic inputs then

Reply:
yes.for the old tests, i leave it at 0.

Currently adding additional tests into the math. 

same for tests in omnipool - all old tests have 0 fee, new tests with non-zero fee are added.

Reply:
but old tests are irelevant with this change

Reply:
i believe it is good to have tests with 0 fee, to ensure that correct calculation of remove liquidity out remains unchanged.

and these tests do exactly this, to ensure that no side effects have been introduced.